src/fiber.cr

```diff
@@ -129,6 +129,7 @@ class Fiber
     @resume_event ||= Crystal::EventLoop.create_resume_event(self)
   end
 
+  # Set the current fiber to wait, in order to eventually execute others fibers.
```

Comment:
This is completely unrelated to documenting Channel. AFAIK developers are encouraged to use `sleep(0)` for the time being anyway.